controllers/admin_client_mock.go

```diff
@@ -397,24 +375,14 @@ func (client *mockAdminClient) IncludeProcesses(addresses []fdbv1beta2.ProcessAd
 	adminClientMutex.Lock()
 	defer adminClientMutex.Unlock()
 
-	newExclusions := make([]string, 0, len(client.ExcludedAddresses))
-	for _, excludedAddress := range client.ExcludedAddresses {
-		included := false
-		for _, address := range addresses {
-			if address.String() == excludedAddress {
-				included = true
-				client.ReincludedAddresses[address.String()] = true
-				break
-			}
+	for _, address := range addresses {
+		address := address.String()
+		_, found := client.ExcludedAddresses[address]
```

Comment:
Most go samples use "ok" in this context but I'm fine with "found", that comment is only informational.